bin/backup-all-my-flickr-photos

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/home/reynolds/xxx/bin/python2
```

Comment:
This change clearly will cause the script to break on any machines but your own. Could it be undone?